core/src/main/resources/schema/app/20220204202948_CB-14399_Modify_unique_CRN_constraint_to_unique_CRN_terminated_constraint.sql

```diff
@@ -0,0 +1,9 @@
+-- // CB-14399: Modify unique CRN constraint to unique (CRN, terminated) constraint.
+-- Migration SQL that makes the change goes here.
+ALTER TABLE stack DROP CONSTRAINT IF EXISTS stack_crn_uq;
+ALTER TABLE stack ADD CONSTRAINT stack_crn_terminated_uq UNIQUE (resourceCrn, terminated);
```

Comment:
this will potentially cause on issue, where it's possible there are multiple terminated stack with the same crn. Repository calls expect single stack as a result and would fail if multiple is returned

Reply:
As resourceCrn's are uniques. The only situation where we see two stacks using the same CRN is the recovery from failed resize. When that happens, one of the stacks should have been terminated.

I agree, with the change we allowing the possibility of having multiple stacks with same CRN in terminated state.

Reply:
No longer doing this, marking this as resolved.